main.cpp

```diff
@@ -0,0 +1,204 @@
+// Configuration editor
+// Command line utiltiy to create configuration files
+// Valid arguments are init and edit
+
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <stdio.h>
+
+
+using std::cout;
+using std::endl;
+using std::string;
+using std::cin;
+
+const string CONFIG_FILE_PATH = "/users/bconway/SourceTree/School/CPT-180-27-Assignment-2-Config-Generator/app.cfg";
+
+
+int main(int argc, char *argv[]) {
+
+    string argument1 = "";
+    // Make sure that the argument isn't empty
+    if (argv[1] != nullptr) {
+        argument1 = static_cast<string>(argv[1]);
+    }
+
+    if (argument1 == "init") {
+
+        // initialize the config file
+        // TODO determine if the file exists and handle the exception
+
+        std::ofstream config_file;
+
+        config_file.open(CONFIG_FILE_PATH);
+
+        // array to hold prompts, setting name and default values
+        const int ARRAY_SIZE = 5;
+        const string SETTING_NAME[ARRAY_SIZE] = {"name", "email", "cypher", "timezone", "knownrecipients"};
+        const string SETTING_PROMPT[ARRAY_SIZE] = {"Enter your name (first and last):", "Enter your email address:",
+                                                   "Enter your unique cypher (password):",
+                                                   "Enter your timezone-offset (example -5:00):",
+                                                   "Enter path to knownrecipients file (hit enter for the default knownrecipients.txt):"};
+        const string SETTING_DEFAULT[ARRAY_SIZE] = {"", "", "", "", "./knownrecipients.txt"};
+
```

Comment:
Really like your use of arrays for this assignment - does the same thing, but with alot less code...and still readable.